test/run-local-e2e.sh

```diff
@@ -62,57 +46,74 @@ pushd "$KURTOSIS_FOLDER" >/dev/null
 log_info "Cleaning any existing Kurtosis enclaves..."
 kurtosis clean --all
 
-# Start Kurtosis Enclave
 log_info "Starting Kurtosis enclave"
-
-if [ "$TEST_TYPE" == "single-l2-network-fork12-op-succinct" ]; then
+case "$TEST_TYPE" in
+single-l2-network-fork12-op-succinct)
     ENCLAVE_NAME="op"
     kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_single_chain_fork12_op_succinct_args.json" .
-elif [ "$TEST_TYPE" == "single-l2-network-fork12-pessimistic" ]; then
+    ;;
+single-l2-network-fork12-pessimistic)
     ENCLAVE_NAME="aggkit"
     kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_single_chain_fork12_pessimistic_args.json" .
-elif [ "$TEST_TYPE" == "multi-l2-networks" ]; then
+    ;;
+multi-l2-networks)
     ENCLAVE_NAME="aggkit"
     kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_multi_chains_args_1.json" .
     kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_multi_chains_args_2.json" .
-else
+    ;;
+*)
     log_error "Unknown test type: $TEST_TYPE"
     exit 1
-fi
-
+    ;;
+esac
 log_info "$ENCLAVE_NAME enclave started successfully."
 popd >/dev/null
 
-if [ "$RUN_TESTS" == "true" ]; then
-    log_info "Using provided Agglayer E2E repo at: $E2E_FOLDER"
+if [ -n "$E2E_FOLDER" ]; then
+    if [ ! -d "$E2E_FOLDER" ]; then
+        log_error "The provided E2E folder does not exist: $E2E_FOLDER"
+        exit 1
+    fi
 
+    log_info "Using provided Agglayer E2E repo at: $E2E_FOLDER"
     pushd "$E2E_FOLDER" >/dev/null
 
-    # Setup environment
     log_info "Setting up e2e environment..."
     set -a
     source ./tests/.env
     set +a
 
+    local imported_bridges_tool="./target/aggsender_find_imported_bridge"
```

Comment:
this will fail as we are not in aggkit folder but e2e folder

Reply:
Fixed (tested in local, everything seems to be working correctly)
https://github.com/agglayer/aggkit/pull/524/commits/9b5302c45b4293afabae09805691d4909200f02d